src/main/kotlin/lotto/view/LottoPurchaseSummaryPrinter.kt

```diff
@@ -3,8 +3,8 @@ package lotto.view
 import lotto.business.LottoTicket
 
 object LottoPurchaseSummaryPrinter {
-    fun print(lottoTickets: List<LottoTicket>) {
-        println("${lottoTickets.size}개를 구매했습니다.")
+    fun print(manualTicketCount: Int, lottoTickets: List<LottoTicket>) {
+        println("수동으로 ${manualTicketCount}장, 자동으로 ${lottoTickets.size}개를 구매했습니다.")
         lottoTickets.forEach { it: LottoTicket -> println(it.lottoNumbers.map { it.number.toString() }) }
```

Comment:
람다식 안의 람다 식의 경우 가급적 `it`이 중첩되니 명시적으로 람다 변수를 다른 값으로 지정해서 사용하거나 지금처럼 메서드 참조를 사용하는 방식으로 우회할 수도 있겠네요. 😃

```suggestion
        lottoTickets.forEach { println(it.lottoNumbers.map(LottoNumber::number)) }
```